src/gui/accelerators.c

```diff
@@ -1210,9 +1218,19 @@ static void _effect_editing_started(GtkCellRenderer *renderer, GtkCellEditable *
 
   const dt_action_element_def_t *elements = _action_find_elements(s->action);
   int show_all = s->action->type != DT_ACTION_TYPE_FALLBACK;
+  int bold_move = _shortcut_is_move(s) ? DT_ACTION_EFFECT_DEFAULT_KEY : DT_ACTION_EFFECT_DEFAULT_DOWN + 1;
   if(elements)
-    for(const gchar **effect = elements[s->element].effects; *effect ; effect++)
-      gtk_list_store_insert_with_values(store, NULL, -1, 0, show_all++ ? _(*effect) : "", -1);
+    for(const gchar **effect = elements[s->element].effects; *effect ; effect++, bold_move++)
```

Comment:
ok a single instruction but multi-line. Please add { and } around gtk_list_store_insert_with_values for readability.